docs/issue_guidance.md

```diff
@@ -0,0 +1,63 @@
+# Issue guidance
+
+This document details how Service Fabric handles issues reported through GitHub.
+
+**Note**, for support, guidance on opening issues, and general questions related to using Service Fabric, take a look
+at the documentation in the [contributing guide](/CONTRIBUTING.md).
+
+# Triaging issues
+
+Issues are triaged using GitHub labels, the following labels are used to indicate the status
+of the issue.
+
+## Bug, enhancement and question
+
+Most labels are used according the GitHub standards, these labels indicate the type of issue.
+
+|     Label     |                                             Description                                             |
+| ------------- | --------------------------------------------------------------------------------------------------- |
+| investigating | The issue isn't clear, further investigation is needed to determine if this a bug or something else |
```

Comment:
I'm okay with users assigning an area if we stress that they shouldn't guess, and instead used `needs triage`.

I think the only other thing that we need is like a `fixing` tag? Thoughts @bpm-ms @vturecek ?

Reply:
@anmolah  I want to make sure that we do not actually "Assign" in Github unless someone is actively working on it or plans to in the near future.  This being due to our previous discussion that we want anyone to potentially pick it up, and we don't want issues assigned to someone prematurely who has no capacity to work on them.  

What about a "needs-investigation" label, and @ mention with a stipulation that anyone @ mentioned should follow up and remove the investigation label? I don't really like the "understood" because it implies a full understanding of what's wrong, and doesn't add much value.  I think removing a "needs-investigation" label could serve a similar purpose.  If the tag is removed, and the area expert has posted their initial findings, it brings the issue into a state most ready to be picked up and actually worked on: 
- Initial investigation completed and info posted in the issue
- Not assigned to anyone, ready for someone to pick it up
- Area assigned, with area owners able to track what active issues exist in their area, and with the ability to assign someone from their team, or themselves to something that needs to be worked on.

I don't like a "fixing" tag, this term only works for bugs.  I think "in-progress" might sound better. 

Basically, I think assignment should only be done if that person specifically intends to allocate capacity to fix the issue soon, and once they start working on it, they can add an in-progress tag.